py/torch_tensorrt/dynamo/backend/__init__.py

```diff
@@ -8,10 +8,10 @@
 from torch_tensorrt import EngineCapability, Device
 from torch_tensorrt.fx.utils import LowerPrecision
 
-from torch_tensorrt.dynamo.backend._settings import CompilationSettings
+from torch_tensorrt.dynamo.common import CompilationSettings
```

Comment:
do you think its worth having dynamo.common.CompilationSettings vs just dynamo.CompilationSettings?

Reply:
I think it would be better as `dynamo.common.CompilationSettings` until _both_ compile and export use them, since `dynamo.CompilationSettings` might imply it is used across `dynamo`, whereas `common` could just be general purpose utilities.